superset/assets/package.json

```diff
@@ -49,7 +49,7 @@
   "dependencies": {
     "@data-ui/sparkline": "^0.0.54",
     "@superset-ui/chart": "^0.9.6",
```

Comment:
Please check versions. `master` has newer version for these `@superset-ui/xxx`

Reply:
This is a branch off of lyftga. I think lyftga is a little behind.

Reply:
oh....haha my bad